group_vars/all

```diff
@@ -31,6 +31,7 @@ marathon_enabled: true
 datadog_enabled: false
 scope_enabled: false
 docker_host: 'unix:///var/run/weave/weave.sock'
+onpremise: false
```

Comment:
I would just remove this flag as it's assuming too much, if it's on premise probably a different solution for every specific case need to be put in place and "defunctzombie.coreos-bootstrap" might not be always needed, or maybe just remove the flag to ansible-coreos-bootstrap or something, as per @tayzlor comment we could move to the conditional to the playbook level